samples/test/auth.test.js

```diff
@@ -0,0 +1,73 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const cp = require('child_process');
+const {assert} = require('chai');
+const {describe, it} = require('mocha');
+const {auth} = require('google-auth-library');
+
+const execSync = (cmd, opts) => {
+  return cp.execSync(cmd, Object.assign({encoding: 'utf-8'}, opts));
+};
```

Comment:
`execFile` or `execFileSync` would be cleaner rather than dealing with spacing, escaping, etc. (I'm assuming this was copied from tests from a few years back). See:
- https://nodejs.org/api/child_process.html#child_processexecfilesyncfile-args-options